Sources/SpeziViews/ViewModifier/ShimmerModifier.swift

```diff
@@ -0,0 +1,68 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+/// A view modifier that applies a shimmer animation effect.
```

Comment:
As the symbol is not public the documentation won't be rendered somewhere; I would suggest to move all of the to the public definition of the modifier.